CONTRIBUTING.md

```diff
@@ -139,6 +140,18 @@
   - Code that requires a rework should not be commented out but completely removed from the code base.
   - If the code is essential for a future fix, please add it to a ticket that explains the problem.
 
+### We Prepare Tickets for the Refinement
+
+- **User Story**: As a core-team member of the Solid Design System, I want to ensure a effective and efficient refinement process, by preparing each ticket with the necessary information during a weekly pre-refinement meeting.
```

Comment:
I would specify more clearly who is part of the pre-refinement and clearly what's the difference to the refinement.